### PR TITLE
fix: (TNLT-1014) Fix author image opening in modal for Sports articles

### PR DESCRIPTION
**Pull request by GeorgianSorinMaxim**
https://nidigitalsolutions.jira.com/browse/TNLT-1014
Open the right image for the author (of sports articles) in the modal on iOS

Before:
Simulator Screen Shot - iPhone 12 - 2020-12-11 at 11 41 29
Simulator Screen Shot - iPhone 12 - 2020-12-11 at 11 41 59

After:
Simulator Screen Shot - iPhone 12 - 2020-12-11 at 11 41 29
Simulator Screen Shot - iPhone 12 - 2020-12-11 at 11 41 39

### CODE REVIEW

File: packages/image/src/modal-image/modal-image.js [hunk 1]
```diff
@@ -54,8 +50,16 @@ class ModalImage extends Component {
...
+      onImagePress,
+      images = [],
+      uri,
+      isAuthorImage,
+    } = this.props;
+
+    if (onImagePress && !isAuthorImage) {
       return (
         <Button onPress={() => onImagePress(index)}>
           <Image {...this.props} />
```
Comment (kamiranoff) [inline packages/image/src/modal-image/modal-image.js:65]: I also changed this in my PR for the images to appear bigger:
https://github.com/newsuk/times-components-native/pull/275/files#diff-684e12edb47a39f0c61d1d235e8dcc0097e9e62832e292d4b372001f1fd79542R127

File: packages/image/src/modal-image/modal-image.js [hunk 2]
```diff
@@ -54,8 +50,16 @@ class ModalImage extends Component {
...
+    const {
+      highResSize,
+      index,
+      onImagePress,
+      images = [],
+      uri,
+      isAuthorImage,
+    } = this.props;
+
+    if (onImagePress && !isAuthorImage) {
```
Comment (c-j-j) [inline packages/image/src/modal-image/modal-image.js:62]: I don't think this modal-image component should be aware of authors (or any other business domain concepts)... can we rename this `isAuthorImage` prop to something more general like `isBasicImage` or `isSingleImage`? That'll make it more reusable

**Comment by gsm-dev-x-ldn** [comment]
I've updated the PR.

File: packages/article-comment-tablet/__tests__/shared.base.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ const emptyArticle = {
   expirableFlags: [],
   label: null,
   standfirst: null,
+  isAuthorImage: false,
```
Comment (c-j-j) [inline packages/article-comment-tablet/__tests__/shared.base.js:24]: `isSmallImage`?

Comment (gsm-dev-x-ldn) [reply packages/article-comment-tablet/__tests__/shared.base.js]: yes, the PR is now updated.

**Comment by aronshamash** [comment]
@GeorgianSorinMaxim I guess this PR title is not quite accurate because we are not just fixing for a article in the Sports section it is about all article template which have a roundel (author) image, correct ?

**Review by kamiranoff — APPROVED** [review]

**Pull request merged**